pub/index.html

```diff
@@ -4,40 +4,154 @@
-                        <i class="fa fa-bomb fa-lg"></i>
-                    </div>
-                    <article class=content-text>
-                        <h3>Title</h3>
-                        <p>Body copy goes here. I need my password and prompt to work!</p>
-                    </article>
+                    <p>Favor packaging over toy. Purr while eating swat at dog. Vommit food and eat it again. Flop over kick up litter eat grass, throw it back up stare at ceiling, but chase dog then run away or scamper. Inspect anything brought into the house vommit food and eat it again hide at bottom of staircase to trip human.</p>
+                    <p>Find something else more interesting. Roll on the floor purring your whiskers off swat at dog. Sun bathe play riveting piece on synthesizer keyboard why must they do that, so rub face on everything hunt anything that moves hiss at vacuum cleaner. </p>
+                    <h3>This is a purrr quote from the article</h3>
+                    <p>Purr for no reason inspect anything brought into the house climb leg, or chase mice poop on grasses. Run in circles hopped up on catnip. Why must they do that leave fur on owners clothes claw drapes cat snacks, or chase laser but sleep in the bathroom sink. Swat at dog give attitude hide from vacuum cleaner jump off balcony, onto stranger's head. Chew iPad power cord under the bed. Vommit food and eat it again all of a sudden cat goes crazy. Swat at dog chase ball of string, yet shove bum in owner's face like camera lens yet need to chase tail. Meowing non stop for food need to chase tail eat grass, throw it back up.</p>
+                    <p>Always hungry who's the baby, for make muffins sweet beast, so play time. Need to chase tail present belly, scratch hand when stroked hopped up on catnip, for chew foot, but attack feet, so swat at dog, so bathe private parts with tongue then lick owner's face.</p>
+                </div>
+            <ul class="button-group">
+                <li class="button-text hidden-mobile">
+                    <i class="fa fa-comment-o fa-lg"></i><a href="#">Read Comments</a>
+                </li>
+                <li class="button-text hidden-mobile">
+                    <i class="fa fa-cloud-upload fa-lg"></i><a href="#">Read Later</a>
+                </li>
+                <li class="button-text hidden-mobile">
+                    <i class="fa fa-share-square-o fa-lg"></i><a href="#">Share This Post</a>
+                </li>
+            </ul>    
+            <div class="button-group">
+                <icon class="fa fa-comment-o fa-lg"></icon>
+                <div class="button-text hidden-mobile">Read Comments</div>
+                <icon class="fa fa-cloud-upload fa-lg"></icon>
+                <div class="button-text hidden-mobile">Read Later</div>
+                <icon class="fa fa-share-square-o fa-lg"></icon>
+                <div class="button-text hidden-mobile">Share This Post</div>
+            </div>
```

Comment:
Swimming slightly ahead of the pack, I see. Hopefully you can see the semantic value of those Bootstrap classes.